doc/man/cockpit-bridge.xml

```diff
@@ -82,6 +82,14 @@
             available to the user running this command.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
```

Comment:
The Closes # still refers to my original PR. Please adjust.

Reply:
Done.

Reply:
No, it's still wrong. :-)